### DISCUSSION TITLE
Preprocessing pipeline for native space GLM

### DISCUSSION THREAD

**Discussion by TamerGezici** [automaticanalysis/automaticanalysis#373] [Q&A] [answered]
Hi, I kindly ask for your suggestions for a plausible pipeline which would analyze a single subject on contrasts within the subject's native space, in the best possible way that utilizes AA's functionality.

I am asking this for a very specific use case, where we estimate contrasts for tumor patients and do functional localization (such as language, memory and other higher cognitive functions)

So far, I had been using the two-step coregisteration algorithm to do this, (ommiting normalization and directly smoothing) but then I noticed this may not be the best approach considering we will not end up normalizing anyway (so the MNI-T1 coregisteration is useless).

**Comment by tiborauer** [comment, marked as answer]
The optimal pipeline depends on the data. If it is only fMRI and you never want to combine across subjects, then a
(fieldmap2VDM - ) realign(unwarp) - slicetiming - coregextended - smooth preprocessing pipeline would be my initial guess. You may want to include segment8 - mask_segment - compSignal to regress out global WM and CSF (but not GM!) signal (first_level_model.compRegs = [2 3]). I would also set first_level_model.moveMat = [1 1 0; 1 1 0] to extended motion correction parameters.
Other parameters mostly depends on the spatial and temporal resolution. Feel free to contact me via other channels for further consultation!
